app/assets/stylesheets/pageflow/themes/default/page.css.scss

```diff
@@ -74,8 +74,7 @@ $page-color: $main-color;
 
 .js .page:first-child {
 
-  .content_and_background .scroller > div:after {
-    content: " ";
+  .logo_header {
```

Comment:
I guess there are 12 properties assigned in this rule, while scsslint thinks there should be at most 10. The limit seems a bit arbitrary. Then again, it is quite a long rule.